controllers/operands/kubevirt.go

```diff
@@ -294,7 +295,71 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+
+	qps, ok := configMap.Data["qps"]
```

Comment:
What's `qps`? if we're using it for a user facing API, maybe we want a human readable name?

Reply:
I agree. I have also used a const for referring these parameters.